samples/symDisplay.c

```diff
@@ -0,0 +1,360 @@
+               set_D1( get_D2() );
+               set_D2( get_D3() );
+               set_D3( get_D4() );
+               set_D4( get_D5() );
+               set_D5( DISP_V_1 );
+               set_D6( DISP_V_2 );
+               break;
+            case 'W':
+               set_D0( get_D1() );
+               set_D1( get_D2() );
+               set_D2( get_D3() );
+               set_D3( get_D4() );
+               set_D4( get_D5() );
+               set_D5( DISP_W_1 );
+               set_D6( DISP_W_2 );
+               break;
+            case 'w':
+               set_D0( get_D1() );
+               set_D1( get_D2() );
+               set_D2( get_D3() );
+               set_D3( get_D4() );
+               set_D4( get_D5() );
+               set_D5( DISP_W_1 );
+               set_D6( DISP_W_2 );
+               break;
+            case 'Y':
+               set_D6( DISP_Y );
+               break;
+            case 'y':
+               set_D6( DISP_Y );
+               break;
+            case 'Z':
+               set_D6( DISP_Z );
+               break;
+            case 'z':
+               set_D6( DISP_Z );
+               break;
+            case ' ':
+               set_D6( DISP_SPACE );
+               break;
+            case '.':
+               set_D6( DISP_PERIOD );
+               break;
+            case '-':
+               set_D6( DISP_HYPHEN );
+               break;
+            case '\'':
+               set_D6( DISP_APOSTR );
+               break;
+            case '"':
+               set_D6( DISP_APOSTR );
+               break;
+            case '=':
+               set_D6( DISP_EQUAL );
+               break;
+            case '_':
+               set_D6( DISP_BOTTOM );
+               break;
+            case '/':
+               set_D6( DISP_SLASH );
+               break;
+            case '\\':
+               set_D6( DISP_BACKSLASH );
+               break;
+            default:
+               displayable = 0;        // Character not mapped
+         }
+
+         if( displayable ) {
+
+            putchar( buffer[l] );      // Send it to the console
+
+            set_D0( get_D1() );        // Scroll to the left
+            set_D1( get_D2() );
+            set_D2( get_D3() );
+            set_D3( get_D4() );
+            set_D4( get_D5() );
+            set_D5( get_D6() );
+
+            for( d = 0; d < flashes ; d++ ) {
+               fdisp();                // Display
+            }
+         }
+      }
+
+      for( e = 0; e < 6; e++ ) {       // Gradually fill the
+         set_D0( get_D1() );           // display with spaces
+         set_D1( get_D2() );
+         set_D2( get_D3() );
+         set_D3( get_D4() );
+         set_D4( get_D5() );
+         set_D5( DISP_SPACE );
+         set_D6( DISP_SPACE );
+         for( d = 0; d < flashes ; d++ ) {
+            fdisp();                   // Display
+         }
+      }
+   }
+
+   puts( "\n\nEnjoy your day!\n\n" );
+
+   return;
+}
```

Comment:
Please change this code to adhere to the cc65 C code style rules - see https://github.com/cc65/wiki/wiki/C-and-Assembly-sources-style-guide

Reply:
I've removed the 'return' line.  Honestly, if these programs were going to run on a computer with a filesystems, I would have always made them return something so I could test for error by the calling process.  But the Sym-1 is more like a microcontroller, and there's nothing that would catch the return value so it makes sense to have main() not return a value.